fastlane/lib/fastlane/actions/setup_ci.rb

```diff
@@ -29,6 +29,11 @@ def self.setup_keychain
           return
         end
 
+        unless Helper.operating_system == 'macOS'
```

Comment:
```suggestion
        unless Helper.mac?
```

Reply:
I also think this should get moved up before the checking of `ENV["MATCH_KEYCHAIN_NAME"]`